libraries/chain/include/eosio/chain/webassembly/eos-vm-oc/intrinsic_mapping.hpp

```diff
@@ -282,6 +279,9 @@ inline constexpr auto get_intrinsic_table() {
       "env.bls_fp_mul",
       "env.bls_fp_exp",
       "env.set_finalizers",
+      "env.call",
+      "env.get_call_data",
+      "env.set_call_return_value",
       "eosvmoc_internal.check_memcpy_params"
```

Comment:
Seems like they should come at the very end.

Reply:
Sorry, I should have been more careful. From `eosvmoc_internal` I just thought that was a last item guard.